packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.pcss

```diff
@@ -2,3 +2,19 @@
 @import '../facet-common.pcss';
 @import '../facet-value-checkbox/facet-value-checkbox.pcss';
 @import '../facet-value-link/facet-value-link.pcss';
+
+/**
+ * @prop --atomic-rating-facet-star-active-color: Color of the star icon when active.
+ */
+.star-active {
+  fill: #f4c315;
+}
+
+/**
+ * @prop --atomic-rating-facet-star-inactive-color: Color of the star icon when inactive.
+ */
+.star-inactive {
+  fill: #e5e8e8;
```

Comment:
Could it be useful to store these colors in the tailwind configuration, and reference them using CSS variables? This would make them easier to customize and theme.

The comments suggest there are CSS variables, but I don't see them being referenced,

Reply:
Yes, I just added the props as `var` and made these the default value for that prop